rustls/src/common_state.rs

```diff
@@ -348,24 +356,29 @@ impl CommonState {
         self.send_appdata_encrypt(data, limit)
     }
 
-    pub(crate) fn start_outgoing_traffic(&mut self) {
+    pub(crate) fn start_outgoing_traffic(
```

Comment:
Here, and for `start_straffic()`, it would be good to add a comment about when/why `sendable_plaintext` is `None`.